app/models/form1010cg/submission_job_claim.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+
+module Form1010cg
+  class SubmissionJobClaim < Common::RedisStore
+    redis_store REDIS_CONFIG[:form_1010_cg_submission_job_claim][:namespace]
+    redis_ttl REDIS_CONFIG[:form_1010_cg_submission_job_claim][:each_ttl]
+    redis_key :claim_id
+
+    attribute :claim_id
+    validates :claim_id, presence: true
+
+    def self.set_key(claim_id)
+      redis_namespace.set(claim_id, 't') unless redis_namespace.exists?(claim_id)
```

Comment:
I was trying to just keep the existing functionality at the moment. This is currently just setting it to `t` and is used to track number of applications with any retries. We also have the `StatsD.increment(#{STATSD_KEY_PREFIX}retries")` that is incremented for each occurrence of a retry, although that isn't linked to specific claim. If we are wanting more data like that I'd probably rather explore the sidekiq datadog gem than try to use this for that. Either way, my main goal of these changes were to ensure we are setting a `ttl` on this redis key and defining it in the `redis.yml`, since when it was originally implemented it did not follow the platform standards defined [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/how-to-guide-caching-with-redis-namespace-in-vets-). If we think other additions would be valuable I'd rather have another ticket and define the goal of what all we want track as opposed to adding more scope to this atm.